demo/device/devicedemo.go

```diff
@@ -44,6 +45,23 @@ func main() {
 	fmt.Println(ietfjson)
 }
 
+// ExampleAnnotation is used to demonstrate the ygot.Annotation interface,
+// and the ability for ygot to add annotations to generated structs.
+type ExampleAnnotation struct {
+	GeneratorSource string `json:"gen-source"`
```

Comment:
(optional) Since this serves as a public example, maybe we shouldn't introduce the Generator term here.

How about a ConfigSource ?

Reply:
SGTM, done.